pygsti/modelmembers/povms/effect.py

```diff
@@ -142,9 +143,12 @@ def frobeniusdist_squared(self, other_spam_vec, transform=None, inv_transform=No
         float
         """
         vec = self.to_dense()
+        other_vec = other_spam_vec.to_dense()
         if transform is not None:
             vec = transform.T @ vec
-        return _ot.frobeniusdist_squared(vec, other_spam_vec.to_dense())
+            if isinstance(inv_transform, _mt.IdentityOperator):
+                other_vec = transform.T @ other_vec
```

Comment:
See comment above.

Reply:
Tentatively resolved by refactor in [this commit](https://github.com/sandialabs/pyGSTi/pull/671/commits/06cff96945bd1025f338b253bff9721bdbe370b8).